Riot/Views/RoomTitle/PreviewRoomTitleView.m

```diff
@@ -106,6 +105,10 @@ - (void)refreshDisplay
         
         // Room topic
         self.roomTopic.text = self.roomPreviewData.roomTopic;
+
+        [UIView setAnimationsEnabled:NO];
+        [self.roomTopic scrollRangeToVisible:NSMakeRange(0, 0)];
```

Comment:
`self.roomTopic.contentOffset = CGPointZero; ` would be simpler

Reply:
except that the rendering is not the same. This screenshot must be compared with the 1st one in the PR description.

The last displayed line is truncated: 

![simulator screen shot - iphone 5s - 2018-03-12 at 17 06 12](https://user-images.githubusercontent.com/8418515/37295284-156591b2-2618-11e8-9797-cbb6baab0c6c.png)